src/pages/ThorChainLP/components/ReusableLpStatus/TransactionRow.tsx

```diff
@@ -239,26 +240,32 @@ export const TransactionRow: React.FC<TransactionRowProps> = ({
   })
 
   useEffect(() => {
-    if (!(txId && tx)) return
+    if (!txId) return
 
-    if (tx?.status === TxStatus.Pending) {
-      setStatus(tx.status)
+    // Avoids this hook's mutate fn running too many times
+    if (status === TxStatus.Confirmed || status === TxStatus.Failed) return
+
+    // Consider rune transactions pending after broadcast and start polling thorchain right away
+    if (isRuneTx) {
+      if (status === TxStatus.Unknown) {
```

Comment:
There's already a `(status === TxStatus.Confirmed || status === TxStatus.Failed)` check above, we might be able to avoid this check - the only two other states are pending (effectively the same as this one) and unknown

Reply:
I removed `setStatus(TxStatus.Pending)` from the `then` clause on the broadcast function so we have a state to key on for rune as we won't be checking the tx.status in this case